src/client/Microsoft.Identity.Client/Internal/Requests/AuthenticationRequestParameters.cs

```diff
@@ -73,7 +73,7 @@ internal class AuthenticationRequestParameters
         public AuthorityEndpoints Endpoints { get; set; }
         public Authority TenantUpdatedCanonicalAuthority { get; set; }
         public ICacheSessionManager CacheSessionManager { get; }
-        public SortedSet<string> Scope { get; }
+        public HashSet<string> Scope { get; }
 
         public bool HasScopes => Scope != null && Scope.Any();
```

Comment:
Suggest to use Scope.Count in place of Scope.Any() to avoid conversion to IEnumerable